spec/Section 3 -- Type System.md

```diff
@@ -1541,6 +1595,21 @@ is constructed with the following rules:
   definition does not provide a default value, the input object field
   definition's default value should be used.
 
+Further, if the input object is a Oneof Input Object, the following additional
+rules apply:
+
+* If the input object literal or unordered map does not contain exactly one
+  entry, an error must be thrown.
+
+* If the single entry in the input object literal or unordered map is {null},
+  an error must be thrown.
+
+* If the coerced unordered map does not contain exactly one entry, an error
```

Comment:
🤔  why do we have basically the same rules twice, once for "unordered map" and once for "coerced unordered map"? Can this be simplified, or are we being explicit about something subtle?

Reply:
See the [note above regarding validation](https://github.com/graphql/graphql-spec/pull/825/files/bb225f77781adfc23df14e391db5ee4a01d1eb6a#r588879692). These new bullets are _coercion_ (rather than validation) constraints; it's the ones above that we might want to remove.